qiskit_experiments/analysis/plotting.py

```diff
@@ -81,18 +88,24 @@ def plot_curve_fit(
     ax.plot(xs, ys_fit, **plot_opts)
 
     # Plot standard error interval
-    if confidence_interval and fit_errors is not None:
+    if monotonicity and fit_errors is not None:
         if param_keys:
             params_upper = {}
             params_lower = {}
             for key, param, error in zip(param_keys, fit_params, fit_errors):
-                params_upper[key] = param + error
-                params_lower[key] = param - error
+                params_upper[key] = param - error * (-1) ** monotonicity[key]
+                params_lower[key] = param + error * (-1) ** monotonicity[key]
```

Comment:
If `params_upper[key] < params_lower[key]` (as is the case when `monotonicity[key] == False`), what is the meaning of "upper" and "lower" here?

Reply:
Always `f(upper) > f(lower)`